content/francais/python-turtle/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Python : Coder avec Turtle"
+description: "Apprenez à dessiner sur un canevas avec Python et Turtle"
+date: 2024-09-17T13:24:17-07:00
+prereq: "Bases de Python"
+difficulty: "Débutant"
+image: ""
+draft: false
+icon: "fab fa-python"
+---
+
+## Introduction
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/uw8_cEooILM" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```

Comment:
I wonder if we can ask the tool to remove youtube embeds? These are not relevant for the translated version as the videos are in English?

Reply:
noted